Update start_your_nodes.md (#2046)
input-output-hk/cardano-node: doc/stake-pool-operations/start_your_nodes.md

    --shelley-vrf-key vrf.skey \
    --shelley-operational-certificate node.cert

> Please note that when running a node, it is important to use process monitoring so that the node can be automatically restarted when it terminates unexpectedly. The node relies on being restarted as part of its mechanism for automatically recovering from disk file corruption or other unexpected conditions. In case the node process is not automatically restarted, the node will not be available to be used by stake pool operators to create blocks or exchange wallet information, for example. 

Reviewer: ```suggestion
Please note that when running a node, it is important to use process monitoring so that the node can be automatically restarted when it terminates unexpectedly. The node relies on being restarted as part of its mechanism for automatically recovering from disk file corruption or other unexpected conditions. In case the node process is not automatically restarted, the node will not be available. Thus for example, stake pools will not be able to create blocks and exchanges will not be able to get wallet information.
```
